tests/integration/test_npm_basic.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import utils
+
+
+def test_npm_basic(test_env, tmpdir):
+    """
+    A basic integration test for the npm package manager.
+
+    Process:
+    * Send new request to the Cachito API
+    * Send request to check status of existing request
+
+    Checks:
+    * Verify that the request completes successfully
+    * Verify that there is a package entry with the name "@reactivex/rxjs",
+        type "npm", and the version "6.6.0"
+    * Verify that there is 1370 dependencies
+    * Verify that the tslib dependency is not a dev dependency (dev key in the dependencies array)
+    * Verify that the other dependencies are dev (dev key in the dependencies array)
+    * Verify that the environment variables "CHROMEDRIVER_SKIP_DOWNLOAD": "true",
+        "CYPRESS_INSTALL_BINARY": "0", "GECKODRIVER_SKIP_DOWNLOAD": "true",
+        and "SKIP_SASS_BINARY_DOWNLOAD_FOR_CI": "true" are set.
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+    initial_response = client.create_new_request(
+        payload={
+            "repo": test_env["package"]["npm"]["repo"],
+            "ref": test_env["package"]["npm"]["ref"],
+            "pkg_managers": test_env["package"]["npm"]["pkg_managers"],
+        },
+    )
+    assert initial_response.status == 201
+
+    completed_response = client.wait_for_complete_request(initial_response)
+    assert completed_response.status == 200
+    assert completed_response.data["state"] == "complete"
+
+    response_package = utils.make_list_of_packages_hashable(completed_response.data["packages"])
+    expected_package = test_env["npm_basic"]["package"]
```

Comment:
I know this test is doing similar checks as the test_valid_data_in_request(for go) but I thought it would be clearer to have variables for each test separately. Can others also comment on this? I would like to know your opinions :-) @akhmanova @lcarva @nirzari

Reply:
ok, I'll move it there and move also the variables in test_env. It makes sense when we move it altogether

Reply:
I agree they have the same idea. My concern is about the first mentioning it is about `npm` and the second not being clear it is about `gomod`. Alternatively, we could rename the data structure for `gomod` in the yaml file to make it clearer that the dependencies and packages refer to `gomod` packages.

Reply:
I also made the changes in yaml file, do you think it's clear enough? @athos-ribeiro @akhmanova